File: kassandra-frontend/src/app/core/services/news.service.ts
import {Injectable} from "@angular/core";
import {NewsProviderService} from "../providers/news/news-provider.service";
import {Observable} from "rxjs";
import {Response} from "@angular/http";

@Injectable()
export class NewsService {

  constructor(private newsProvider: NewsProviderService) {
  }

  public getNewsByDate(date: string): Observable<Response> {
    return this.newsProvider.getNewsByDate(date);
  }
}

File: kassandra-backend/src/main/java/com/kassandra/service/news/NewsService.java
package com.kassandra.service.news;

import com.kassandra.repository.domain.News;
import com.kassandra.service.Service;

import java.util.Date;
import java.util.List;

public interface NewsService extends Service<News> {
    public List<com.kassandra.repository.domain.News> getNewsByDate(Date date);
}
